packages/appstore/src/assets/wallets/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import IntroducingWallets from 'Assets/wallets/introducing-wallets.svg';
+import IntroducingWalletsEU from 'Assets/wallets/introducing-wallets-eu.svg';
+import HowItWorks from 'Assets/wallets/how-it-works.svg';
+import TradingAccounts from 'Assets/wallets/trading-accounts.svg';
+import { IconProps } from '../svgs/icon-types';
+
+export const PlatformIcons = {
```

Comment:
Since you can reuse your banner, i think creating a component like this is a better approach 🙏

Reply:
@hamid-deriv @thisyahlen-deriv My suggestion is to leave `Icon` component (maybe move it to components folder in a future) and add one more component for Image and name it, for example, `WalletsImage` instead of `TradingPlatformIcon` with the own props instead of `IconProps` and put there all images for wallets

Reply:
@thisyahlen-deriv please check the PR for this card and fallow the same approach

Reply:
done @mahdiyeh-deriv